cypress/e2e/04-covid_only_tests.cy.js

```diff
@@ -0,0 +1,242 @@
+      "exist",
+    );
+
+    cy.contains("label", "When did the patient's symptoms start?")
+      .next("input")
+      .type("2021-10-05");
+
+    cy.contains("legend", "Select any symptoms the patient is experiencing")
+      .next("div")
+      .within(() => {
+        cy.contains("label", "Chills").click();
+        cy.contains("label", "Headache").click();
+      });
+
+    cy.checkAccessibility();
+
+    cy.contains("Submit results").click();
+    cy.wait("@SubmitQueueItem");
+
+    cy.contains(`Result for ${patientName} was saved and reported.`);
+    cy.get(".prime-home .grid-container").should("not.have.text", patientName);
+
+    cy.get("#desktop-results-nav-link").click();
+    cy.get(".usa-table").contains(patientName);
+
+    // // stores the patient link
+    cy.get(".sr-test-result-row").then(($row) => {
+      const dataTestId = $row.attr("data-testid");
+      const testEventId = dataTestId.split("-").slice(2, 7).join("-");
+
+      cy.task("setTestEventId", testEventId);
+      cy.task("setPatientName", patientName);
+    });
+
+    cy.checkAccessibility();
+  });
+
+  it("starts a test via the manage patients page action menu and then submits a test", () => {
+    cy.visit("/");
+    cy.get(".usa-nav-container");
+    cy.get('[data-cy="desktop-patient-nav-link"]').click();
+    cy.get('[data-cy="manage-patients-header"]').contains("Patients");
+    cy.get('[data-cy="manage-patients-header"]').contains("Showing");
+    cy.get('[data-cy="manage-patients-search-input"]').type(lastName);
+    cy.wait("@GetPatientsByFacility");
+    cy.get('[data-cy="manage-patients-page"]').contains(patientName);
+
+    cy.contains("tr", patientName).find(".sr-actions-menu").click();
+    cy.contains("Start test").click({ force: true });
+
+    cy.wait("@GetFacilityQueue", { timeout: 20000 });
+    cy.contains("Newly added patients go to the bottom of the queue").click();
+
+    cy.contains("legend", "COVID-19 result")
+      .next("div")
+      .within(() => {
+        cy.contains("label", "Negative (-)").click();
+      });
+
+    cy.contains("legend", "Is the patient pregnant?")
+      .next("div")
+      .within(() => {
+        cy.contains("label", "No").click();
+      });
+
+    cy.contains("legend", "Is the patient currently experiencing any symptoms?")
+      .next("div")
+      .within(() => {
+        cy.contains("label", "No").click();
+      });
+
+    cy.contains("Submit results").click();
+    cy.wait("@SubmitQueueItem");
+  });
+
+  it("starts and submits a test for a new patient from the patient form", () => {
+    const patient = generatePatient();
+    patient.email = "myemail@test.com";
+
+    cy.visit("/");
+    cy.get(".usa-nav-container");
+    cy.get("#desktop-patient-nav-link").click();
+    cy.get("#add-patient").click();
+    cy.get("#individual_add-patient").click();
+    cy.get(".prime-edit-patient").contains("Add new patient");
+    cy.injectSRAxe();
```

Comment:
I believe we need to only call `cy.injectSRAxe()` once per file

Reply:
Removing the second injection broke the tests, so it looks like you need it once per `it` block. Let me know if I'm missing something though.

Reply:
ahhh oops I think I mis-remembered and it might be once per `it` block 😅 sorry about that!